apps/nuxt3-ssr/components/CollectionEventDisplay.vue

```diff
@@ -1,6 +1,5 @@
 <script setup lang="ts">
 import collectionEventGql from "~~/gql/collectionEvent";
-import ContentBlockModal from "./content/ContentBlockModal.vue";
```

Comment:
hmm, in intellij this results in error (at least it did a few months ago). 
Don't know if there is plugin for that?

Applies to all other cases too

Don't get me wrong, happy to see this go :-)